src/App.tsx

```diff
@@ -35,61 +36,118 @@ export function getReorderedGoods(
   const visibleGoods = [...goods];
 
   // Sort and reverse goods if needed
+  visibleGoods.sort((goodInit, goodNext) => {
+    switch (sortType) {
+      case SortType.ALPABET:
+        return goodInit.localeCompare(goodNext);
+      case SortType.LENGTH:
+        return goodInit.length - goodNext.length;
+      case SortType.NONE:
+      default:
+        return 0;
+    }
+  });
+
+  if (isReversed) {
+    visibleGoods.sort(() => -1);
```

Comment:
you can use separate array method => reverse()

Reply:
agree